package.json

```diff
@@ -25,7 +26,7 @@
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.6",
-    "clean-webpack-plugin": "^3.0.0",
+    "clean-webpack-plugin": "^1.0.1",
```

Comment:
why did clean-webpack-plugin@3 break things?

Reply:
Between the latest clean-webpack-plugin and copy-webpack-plugin, copy doesn’t actually copy assets on every compilation. Not sure who’s at fault here, but here’s a thread on it: https://github.com/webpack-contrib/copy-webpack-plugin/issues/385